adapter-http/src/main/resources/activities/documents-api/http-docsapi-crud-dataset.yaml

```diff
@@ -7,11 +7,12 @@ description: |
 
 scenarios:
   default:
-    schema:   run driver=http tags==block:"schema.*" threads==1 cycles==UNDEF
-    write:    run driver=http tags==name:'write.*' cycles===TEMPLATE(write-cycles,TEMPLATE(docscount,10000000)) threads=auto errors=timer,warn
-    read:     run driver=http tags==name:'read.*' cycles===TEMPLATE(read-cycles,TEMPLATE(docscount,10000000)) threads=auto errors=timer,warn
-    update:   run driver=http tags==name:'update.*' cycles===TEMPLATE(update-cycles,TEMPLATE(docscount,10000000)) threads=auto errors=timer,warn
-    delete:   run driver=http tags==name:'delete.*' cycles===TEMPLATE(delete-cycles,TEMPLATE(docscount,10000000)) threads=auto errors=timer,warn
+    schema: run driver=http tags==block:"schema.*" threads==1 cycles==UNDEF
+    main: run driver=http tags==block:"main.*" cycles===TEMPLATE(main-cycles,TEMPLATE(docscount,10)) threads=auto errors=timer,warn
+    # write:    run driver=http tags==block:"write.*" cycles===TEMPLATE(write-cycles,TEMPLATE(docscount,10)) threads=auto errors=timer,warn
```

Comment:
Are the supposed to be commented out?

Reply:
Yes, leaving for historical reference as those don't appear to be used at the moment.